src/content/core/remote-execution.mdx

```diff
@@ -25,17 +25,13 @@ deploy:
 Remote execution is always enabled for `okteto test`
 :::
 
-By default, your commands will run in a remote container using our [default container image](https://github.com/okteto/pipeline-runner).
+By default, your commands will run in a remote container using our [default container image](https://github.com/okteto/pipeline-runner) in the workdir `/okteto/src`.
 The default container image is a Debian Linux container with the following tools preinstalled:
 
 - `bash`
-- `cue`
```

Comment:
This is in preparation to remove some of these tools and make it easier to maintain the runner image